doc/man7/des_modes.pod

```diff
@@ -16,7 +16,7 @@ other things.
 
 Normally, this is found as the function I<algorithm>_ecb_encrypt().
 
-=over 2
+=over 4
```

Comment:
This results in 3 spaces between the "*" and the text, which I think is for no good reason.